core/persistence/README.md

```diff
@@ -0,0 +1,11 @@
+# @baseline-protocol/persistence
```

Comment:
If a document/record is baselined, will there be a completely new record that tracks the baseline state and has a one-way reference to the original document? Or, will the original record in the user's database be modified to add new baseline relevant data fields?

Reply:
That is a design decision at the "application level" for now.

I can see both approaches you mentioned being valid and coming with trade-offs. This will be a good follow-up for the persistence package.

We are currently doing the latter for the sake of simplicity. The merkle tree db consists of arguably "new records" to track the state; we should look into how to index the leaves in the tree so the state of a document can be retrieved for any arbitrary/historic state transition.